### PR TITLE
Added missing property 'local' in Gateway's peer response

### PR DESCRIPTION
**Pull request by mharkus**
This commit will add the 'local' property in the /gateway's sample response. Although I don't know the description of it.

### CODE REVIEW

File: doc/api/Gateway.md [hunk 1]
```diff
@@ -53,7 +53,10 @@ returns information about the gateway, including the list of connected peers.
         // inbound is true when the peer initiated the connection. This field
         // is exposed as outbound peers are generally trusted more than inbound
         // peers, as inbound peers are easily manipulated by an adversary.
-        "inbound":    Boolean
+        "inbound":    Boolean,
+
+        // what does local mean?
+        "local":      Boolean
```
Comment (petabytestorage) [inline doc/api/Gateway.md]: connected through the `loopback` interface

Comment (lukechampine) [inline doc/api/Gateway.md]: the precise definition is here: https://github.com/NebulousLabs/Sia/blob/master/modules/netaddress.go#L63

So "local" can mean another node on your LAN, e.g. `10.0.0.5`, not just another node on the same computer

Comment (petabytestorage) [inline doc/api/Gateway.md]: good point. those "private ip address ranges" are from rfc 1918 (v4) and rfc 4193 (v6).

**Comment by lukechampine** [comment]
thanks!

**Pull request merged**